nipype/algorithms/tests/test_modelgen.py

```diff
@@ -12,10 +12,18 @@
 import pytest
 import numpy.testing as npt
 from nipype.interfaces.base import Bunch, TraitError
-from nipype.algorithms.modelgen import (SpecifyModel, SpecifySparseModel,
-                                        SpecifySPMModel)
+from nipype.algorithms.modelgen import (bids_gen_info, SpecifyModel,
+                                        SpecifySparseModel, SpecifySPMModel)
 
 
+def test_bids_gen_info():
+    fname = os.path.realpath('../../testing/data/events.tsv')
```

Comment:
```suggestion
    fname = os.path.realpath('data/events.tsv')
```

Reply:
ah, I see I had been running the tests from another directory..

Reply:
I still get
```
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest-of-neuro/pytest-0/popen-gw0/test_warning0/data/events.tsv'
```